omnibus/package-scripts/agent/postinst

```diff
@@ -55,6 +55,27 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
             ;;
         esac
         #DEBHELPER#
+
+        # Set the installation information if not already present;
+        # This is done in posttrans for .rpm packages
```

Comment:
Why? Because posttrans is an rpm only step?

Reply:
I don't know, this was already on your original PR: https://github.com/DataDog/datadog-agent/pull/5481/files#diff-cd15ba795b31a5f33c05aeac679f34414ef68404078ceb6c786c1a0cfb51e0deR59-R60

Reply:
Ah, old me says

> The rpm install_info is created in posttrans to avoid creating the file until the last moment, since rpm does not seem to recover from failed installs and we wouldn't want to leave the file laying around in that case.

seems reasonable.

Reply:
It's great that you documented it back then 😃